src/wh_nvm.c

```diff
@@ -87,6 +87,36 @@ int wh_Nvm_GetAvailable(whNvmContext* context,
             out_reclaim_size, out_reclaim_objects);
 }
 
+int wh_Nvm_AddObjectPessimistic(whNvmContext* context, whNvmMetadata *meta,
+    const uint8_t* data, whNvmSize dataLen)
```

Comment:
Keep the same argument order
```suggestion
    whNvmSize dataLen, const uint8_t* data)
```

Reply:
I'll change it for now but I think all the functions should use buffer, length (like all the woflCrypt, wolfssl, wolfBoot etc. functions)